model/websocket_client.go

```diff
@@ -20,15 +20,15 @@ const (
 // communicate with a websocket endpoint.
 type WebSocketClient struct {
 	Url                string                  // The location of the server like "ws://localhost:8065"
-	ApiUrl             string                  // The api location of the server like "ws://localhost:8065/api/v3"
+	ApiUrl             string                  // The Api location of the server like "ws://localhost:8065/api/v3"
 	ConnectUrl         string                  // The websocket URL to connect to like "ws://localhost:8065/api/v3/path/to/websocket"
-	Conn               *websocket.Conn         // The WebSocket connection
-	AuthToken          string                  // The token used to open the WebSocket
+	Conn               *websocket.Conn         // The webSocket connection
+	AuthToken          string                  // The token used to open the webSocket connection
 	Sequence           int64                   // The ever-incrementing sequence attached to each WebSocket action
 	PingTimeoutChannel chan bool               // The channel used to signal ping timeouts
-	EventChannel       chan *WebSocketEvent    // The channel used to receive various events pushed from the server. For eg: typing, posted.
+	EventChannel       chan *WebSocketEvent    // The channel used to receive various events pushed from the server. For example: typing, posted
 	ResponseChannel    chan *WebSocketResponse // The channel used to receive responses for requests made to the server
-	ListenError        *AppError               // This field will be set if there was an abnormal closure of the websocket
+	ListenError        *AppError               // This field will be set if there was an abnormal closure of the websocket connection
```

Comment:
I would probably rephrase into `A field that is set if ...`  or `An error that is set if ...` so we keep the same structure.

Reply:
Going with the formet.